### ISSUE TITLE
Lab 3 Add-20533Environment module not working

### ISSUE THREAD

**Issue by mdutkiewicz** [MicrosoftLearning/20533-ImplementingMicrosoftAzureInfrastructureSolutions#365]
We are in module 3, and the add-20533Environment for MOdule 3 is not working, i.e. the script stops without any explanation, as a result no Resource Group is created and therefore no ressource neither.
What's missing ? We select Region 5 (West Europe), where we are located (from Switzerland).

Thanks for your precious help :-),

**Comment by polichtm** [comment]
Are you able to run successfully the provisioning scripts for other modules? Btw. pls note that its name is Add-20533EEnvrionment (double E)
rgds
Marcin

**Comment by mdutkiewicz** [comment]
Yes other modules are provisionning successfully with the same module cmdlet

**Comment by polichtm** [comment]
The provisioning script is supposed to generate a transcript - can you provide its copy?
If you don't see it being generated - can you open the script in the Windows PowerShell ISE, run it line by line, and post the output?
rgds
Marcin
